knowledge-base/common-telerik-themes-libman.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Using Telerik Blazor Themes with LibMan
+description: Learn how to load and update Telerik themes to your Blazor app with the LibMan client library tool.
+type: how-to
+page_title: How to use Telerik Blazor Themes with LibMan
+slug: common-kb-telerik-themes-libman
+position: 
+tags: telerik, blazor, layout, bootstrap
+ticketid:
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>UI for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+This KB article answers the following questions:
+
+* How to use LibMan to add Telerik themes to your Telerik Blazor app.
+* How to update local themes in `wwwroot` automatically when there is a new version.
+
+[LibMan](https://learn.microsoft.com/en-us/aspnet/core/client-side/libman/) is a client-side dependency manager, which is built-in ASP.NET Core. You can use it to automatically download a Telerik theme to the `wwwroot` folder of your app, instead of using [static NuGet assets]({%slug themes-built-in%}#loading-themes-from-the-nuget-package) or [Telerik themes CDN]({%slug themes-swatches%}#swatch-urls).
```

Comment:
"built-in" requires an additional preposition (in/into) which causes repetition
```suggestion
[LibMan](https://learn.microsoft.com/en-us/aspnet/core/client-side/libman/) is a client-side dependency manager, which is integrated into ASP.NET Core. You can use it to automatically download a Telerik theme to the `wwwroot` folder of your app, instead of using [static NuGet assets]({%slug themes-built-in%}#loading-themes-from-the-nuget-package) or [Telerik themes CDN]({%slug themes-swatches%}#swatch-urls).
```